framework/src/application/node/synchronizer/utils.ts

```diff
@@ -117,7 +117,7 @@ export const computeBlockHeightsList = (
 	listSizeLimit: number,
 	currentRound: number,
 ): number[] => {
-	const startingHeight = (currentRound - 1) * activeDelegates;
+	const startingHeight = Math.max((currentRound - 1) * activeDelegates, 0);
```

Comment:
In earlier review, it was like that, but we discussed that this value will never gonna be below zero. So which triggers to put back this logic?

Reply:
when `currentRound` is 0, then `0-1 * 103` will be `-103` and this happens due to `this.dpos.rounds.calcRound(
			this._chain.lastBlock.header.height,
		)` for height 0